File: 백준/Bronze/2775. 부녀회장이 될테야/부녀회장이 될테야.py
import sys
input = sys.stdin.readline


for _ in range(int(input())):
    k = int(input())
    n = int(input())

    floor = [i for i in range(1, n+1)]

    for _ in range(k):
        for i in range(n):
            if i == 0:
                floor[i] = 1
                continue
            else:
                floor[i] = floor[i]+floor[i-1]

    print(floor[-1])

File: 백준/Bronze/2775. 부녀회장이 될테야/부녀회장이 될테야.swift
let casenum = Int(readLine()!)!

for _ in 0..<casenum {
    var k = Int(readLine()!)!
    var n = Int(readLine()!)!
    var floor = Array(1...n)
    
    for _ in 0..<k {
        
        for i in 0..<n{
            if i == 0 {
                continue
            } else {
                floor[i] = floor[i] + floor[i-1]
            }
        }
    }
    
    print(floor.last!)
}
